packages/lime-system/files/usr/lib/lua/lime/utils.lua

```diff
@@ -176,10 +175,18 @@ function utils.has_value(tab, val)
 	return false
 end
 
---! contact array t2 to the end of array t1
-function utils.arrayConcat(t1,t2)
-	for _,i in ipairs(t2) do
-		table.insert(t1,i)
+--! contact array a2 to the end of array a1
+function utils.arrayConcat(a1,a2)
+	for _,i in ipairs(a2) do
+		table.insert(a1,i)
+	end
+	return a1
+end
+
+--! melt table t1 into t2, if keys exists in both tables use value of t2
+function utils.tableMelt(1,t2)
```

Comment:
This seems a fatal type 1 instead of t1 ;)

Reply:
man you're picky today ;)